public/javascripts/AccessibilityChoropleth.js

```diff
@@ -179,13 +179,23 @@ function AccessibilityChoropleth(_, $, turf, difficultRegionIds) {
             layers.push(layer);
 
             layer.on('mouseover', function (e) {
+                for (var i = layers.length - 1; i >= 0; i--) {
+                    if (currentLayer !== layers[i]) {
+                        layers[i].setStyle({opacity: 0.25, weight: 1});
+                    }   
+                }
                 this.setStyle({opacity: 1.0, weight: 3, color: "#000"});
-                this.openPopup()
+                this.openPopup();
             });
             layer.on('mouseout', function (e) {
-                for (var i = layers.length - 1; i >= 0; i--) {
-                    if (currentLayer !== layers[i])
-                        layers[i].setStyle({opacity: 0.25, weight: 1});
+                var mouseoutTrigger = e.originalEvent.toElement.className;
+                if (mouseoutTrigger != "leaflet-popup-content" && mouseoutTrigger != "leaflet-popup-content-wrapper" && mouseoutTrigger != "region-selection-trigger") {
```

Comment:
And are all three of those class names options? Is it sometimes one and sometimes the other? Just curious where all those different class names come from.

Reply:
leaflet-popup-content-wrapper is the whole popup container which can be triggered depending on where the mouse is. leaflet-popup-content is slightly inside that and can also be triggered if its inside the wrapper. region-selection-trigger is the link that is within each region, which can also be triggered if the mouse goes directly to that point.

Reply:
Nice, thank you!